deploy/template/usr/local/bin/start-docker-worker

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+DOCKER_WORKER='node /home/ubuntu/docker_worker/src/bin/worker.js'
+DOCKER_WORKER_OPTS=
+META='http://170.254.169.254/latest/meta-data/'
```

Comment:
looks like this is unused?

Reply:
Hrm, just to make it equal to its `upstart` counterpart.

Reply:
Does it need `export` then?

Reply:
It the original it hadn't